rand_distr/src/gamma.rs

```diff
@@ -650,6 +699,24 @@ where
             })
         }
     }
+
+    /// Returns the alpha shape (`alpha`) of the distribution.
+    pub fn alpha(&self) -> F {
+        if self.switched_params {
+            return self.b;
+        }
+
+        self.a
```

Comment:
Nit: early `return` is poor code style where it's not used to skip further computations.

Reply:
Do you prefer to:
1. Use an intermediate assignment upon checking the condition,
2. Use a rust-like approach with a match,
3. Other?

Is there any documentation about the coding standards that are used in this repository? I don't see any `CONTRIBUTING.md` guide.

Reply:
I would just use `if .. { self.b } else { self.a }`. Simple.

No, we don't have a style guide beyond *make it neat*. https://rust-random.github.io/book/contributing.html

Reply:
Sounds good! I'll update the getter!